language/move-prover/stackless-bytecode-generator/src/function_target_pipeline.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    function_target::{FunctionTarget, FunctionTargetData},
+    stackless_bytecode_generator::StacklessBytecodeGenerator,
+};
+use itertools::Itertools;
```

Comment:
Where does this get used?

Reply:
collect_vec, join, and a few others. I prefer collect_vec for collect because it works better during development when the program is not yet type correct.